internal/ingest/ingest.go

```diff
@@ -121,6 +151,16 @@ func NewIngester(cfg config.Ingest, h host.Host, idxr indexer.Interface, reg *re
 		return nil, err
 	}
 
+	ing.toStaging, ing.cancelOnSyncFinished = ing.sub.OnSyncFinished()
```

Comment:
Ah yes, good catch! With the recent change that adds the custom depth, this would have been broken. I'm not sure if we want to change those go-legs semantics, we should think about it. But for the mean time I can use a scoped block hook to create my own SyncFinishedEvent and call `runIngestStep` manually.

Reply:
It probably makes the most sense to be able to pass a flag that updates the latest even with a selector that's passed in.

Reply:
Short term fix here: f069cd5